### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by cbosdo**
Tiny fixes:

* Add sumaform tools for SLES 15SP1 to minima config
* Fix taskomatic debug options after tanuki drop

### CODE REVIEW

File: salt/suse_manager_server/taskomatic.sls [hunk 1]
```diff
@@ -4,15 +4,10 @@ include:
...
-  file.append:
-    - name: /usr/share/rhn/config-defaults/rhn_taskomatic_daemon.conf
-    - text: |
-        wrapper.java.additional.7=-Xdebug
-        wrapper.java.additional.8=-Xrunjdwp:transport=dt_socket,address=8001,server=y,suspend=n
-        wrapper.java.additional.9=-Dcom.sun.management.jmxremote.port=3334
-        wrapper.java.additional.10=-Dcom.sun.management.jmxremote.ssl=false
-        wrapper.java.additional.11=-Dcom.sun.management.jmxremote.authenticate=false
-        wrapper.java.additional.12=-Djava.rmi.server.hostname={{ salt['network.interfaces']()['eth0']['inet'][0]['address'] }}
+  file.replace:
```
Comment (moio) [inline salt/suse_manager_server/taskomatic.sls]: This has to be guarded with an if that checks the product version

Comment (cbosdo) [reply salt/suse_manager_server/taskomatic.sls]: you're right... I tend to forget that we may have older versions ;)

Comment (cbosdo) [reply salt/suse_manager_server/taskomatic.sls]: :heavy_check_mark:

**Comment by cavalheiro** [comment]
@cbosdo this needs a fix: JMX settings for taskomatic are using a hardcoded IP address: `-Djava.rmi.server.hostname=192.168.122.94`

**Pull request merged**